<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSignPage</class>
 <widget class="QDialog" name="MultiSignPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>791</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>合成多签地址</string>
   </property>
   <widget class="QLineEdit" name="importLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="importLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>钱包地址</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>合成多签地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>多签地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="multisigAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="satAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="satAddressLabel">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>卫星公钥</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdAddressLineEdIt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="thirdAddressLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>用户公钥</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>791</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>多签转帐</string>
   </property>
   <widget class="QLabel" name="toAddresLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>收款地址</string>
    </property>
   </widget>
   <widget class="QLabel" name="transferLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>转帐金额</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="toAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="signButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签名</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dataTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>591</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="signDataLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>签名数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emailButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLabel" name="fromAddressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>多签地址</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fromAddressLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>791</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>广播卫星hex码</string>
   </property>
   <widget class="QTextEdit" name="rawHexTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="broadcastButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>广播</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
